### DISCUSSION TITLE
Adding a network drive with the photoprism sample docker-compose file

### DISCUSSION THREAD

**Discussion by am1rassad1** [photoprism/photoprism#3868] [General]
Hope any1 has the time to help out a total noob. 

I tried for 2 days to add a my NAS / Network drive to the library and it doesnt work. I already searched and found some solutions but what keeps me of accomplishing is that they didnt use the sample docker-compose file which photoprism provides. 

This is the section of the volumes of the sample docker-compose file: 

` ## Storage Folders: use "/" not "\" as separator, "~" is a shortcut for C:/user/{username}, "." for the current directory
    volumes:
      # "C:/user/username/folder:/photoprism/folder"       # example
      - "I:/Bilder:/photoprism/originals"                 # original media files (photos and videos)
      # - "D:/example/family:/photoprism/originals/family" # *additional* media folders can be mounted like this
      # - "E:/:/photoprism/import"                         # *optional* base folder from which files can be imported to originals
      - "./storage:/photoprism/storage"                    # *writable* storage folder for cache, database, and sidecar files (never remove)`

How do I add here an network drive? I already tried the network drive letter and the ip adress - but that doesnt work.

**Comment by lastzero** [comment]
Have you already checked our troubleshooting guide? While the drive letter should work (if you mounted the network drive on your Windows PC), you can also try the following:

- https://docs.photoprism.app/getting-started/troubleshooting/docker/#network-storage
